Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -100,16 +95,20 @@ extension NotificationsViewController: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: requestCellIdentifier) as! RequestTableViewCell
+        
         cell.configure(request: allRequests[indexPath.row])
         return cell
     }
     
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         // TODO: Make this dynamic. Works ok for now, but .automaticDimension didn't really work.
         if let approved = allRequests[indexPath.row].approved {
-            return approved ? 100 : 110
+            if NetworkManager.shared.currentUser.id == allRequests[indexPath.row].approver.id && !approved {
+                return 110
+            }
         }
         return 100
     }
     
```

Comment:
I remember having to make a cell dynamic one time. You can set a constraint for every side so that it will hug the view properly. If that doesn't work then you may have to find a way to calculate a fixed height (seems like that's what you did), but make sure it works for different screen sizes.